src/js/components/FormField/FormField.js

```diff
@@ -236,6 +241,16 @@ const FormField = forwardRef(
       (themeBorder &&
         children &&
         Children.map(children, (child) => {
```

Comment:
I have few questions over this code:
- Why is themeBorder is added in this condition over here ?
-  Is there a sceneario where multiple children are required for one FormField component ?

Reply:
Good questions!

> Why is themeBorder is added in this condition over here ?

This code checks if a border for formFields is defined in theme. If so, any border on the component that is a child of FormField should be removed and instead a border should be placed on the FormField accordion to the specification defined in `theme.formField.border`.

> Is there a sceneario where multiple children are required for one FormField component ?

There wasn't a case I could think of where we would end up with multiple children for FormField. I assume we must have run into this case somewhere though based on the map function that is being used.